packages/cli/src/fingerprint/fingerprintWatchCommand.ts

```diff
@@ -56,8 +56,13 @@ export default class FingerprintWatchCommand {
     return errorMessage.substring(quoteStartIndex + 1, quoteEndIndex);
   }
 
+  isError(error: unknown, code: string): boolean {
+    const err = error as NodeJS.ErrnoException;
+    return err.code === code;
+  }
+
   async watcherErrorFunction (error: Error) {
-    if (this.watcher && error.message.includes("ENOSPC: System limit for number of file watchers reached")) {
+    if (this.watcher && this.isError(error, "ENOSPC")) {
```

Comment:
Did you run prettier? I don't think it allows "" quotes.

Make sure to set up you git hooks to get automatic code formatting before commit. Alternatively, set up prettier extension in vscode to automatically format on save.

Reply:
I didn't run prettier.  I ran `yarn run lint --fix`.

Reply:
That's not enough. You need to run prettier. Lint fix just fixes lint problems (probably not a good idea anyway to do it automatically), prettier is for formatting. You should set up husky or the vscode extension so formatting gets applied automatically.